index.bs

```diff
@@ -614,6 +614,30 @@ information about the platform version on which a given [=user agent=] is execut
 [=Structured Header=] whose value MUST be a [=structured header/string=]
 [[!RFC8941]].
 
+[=User agents=] SHOULD return a version with a platform-specific format that allows differentiating significant platform versions by running these steps:
+    1. Let |platformVersionComponentList| be a [=list=].
+    1. Use per-platform logic:
+        * If the platform is Android:
+            1. Let |platformReturnedVersionString| be the result of querying the OS's `android.os.Build.VERSION.RELEASE` string.
+            1. Parse |platformReturnedVersionString|, storing the first three decimal integers delimited by U+002E (FULL STOP). If |platformReturnedVersionString| does not contain three integer values, replace the missing components with "0".
```

Comment:
Since this is used in 3 places, it's probably useful to pull out a `"Parse the platform returned version string"` `<dfn>` here (modulo a better name), which can take a `platformReturnedVersionString` as an argument (presumably https://infra.spec.whatwg.org/#strictly-split (on 0x2E), then do some kind of `parseInt` operation (maybe?), grab the first 3 ints or zeros otherwise). 

If it returns a list of the 3 version components we care about (or 0s), you could then set `platformVersionComponentList` to the return value in the next step.

Reply:
I tried to review other Bikeshed docs and didn't see much about parsing things-- I think it's likely fine to just call out that it must be an unsigned integer.